src/phoenix/server/app.py

```diff
@@ -87,6 +87,7 @@
 from phoenix.server.grpc_server import GrpcServer
 from phoenix.server.telemetry import initialize_opentelemetry_tracer_provider
 from phoenix.trace.schemas import Span
+from phoenix.utilities.client import PHOENIX_SERVER_VERSION_HEADER
```

Comment:
Can we avoid this import to keep server and client easily separable?

Reply:
it's easy to separate either way, it's a constant, this is just a way to coordinate on the name. I'd argue that on the hierarchy of couplings we need untangle when making new packages a single name import has never made separating work into a package more or less difficult